### ISSUE TITLE
Cables resistance and converter model

### ISSUE THREAD

**Issue by marcin212** [Electrical-Age/ElectricalAge#316]
## Cables resistance

Cables in this mod have way too much resistance. In reality such resistance values are reached over 1km, not one meter.
We could also use cables that could handle higher voltages, like 10, 110 and 220 kV. Not for machines, just for long-distance transport.
Thanks to already existing transformers, achieving such voltages is already possible. I understand that balance could be an issue - everybody would use extreme voltage cables and not worry about overvoltage. This could be balanced out in two ways. One is already implemented - exploding machines. Secondly, they could be adequately high priced.
An example of such cables is in my fork: https://github.com/marcin212/ElectricalAge/blob/Starchasers/src/main/java/mods/eln/Eln.java#L1166-L1375 .
I referenced from these real-life cables: 
- http://www.tfkable.com/sites/default/files/katalogi/tfkable_katalog_kable_wysokich_napiec_pl_2012.pdf
- http://www.tfkable.com/sites/default/files/katalogi/Kable_i_przewody_elektroenergetyczne.pdf

Resistance is calculated by the cross section and length (1 block = 1m) and cable material (resistivity is based on real world values).

Long story short - could you review a potential PR like this and tell me what you think of it?
## Converter model

Second thing - the look of the converter. My buddy made a model that looks like a high power plug: 

![](http://img.bymarcin.com/screenshot-143228-031115.png)
![](http://img.bymarcin.com/screenshot-143143-031115.png)
![](http://img.bymarcin.com/screenshot-021431-061115.png)
![](http://img.bymarcin.com/screenshot-021357-061115.png)
Would you be interested in this?

**Comment by bloxgate** [comment]
@Baughn is working on power poles that can transmit long distance power in the kilovolt range. Perhaps you two could work together on higher voltage power transmission?

**Comment by Baughn** [comment]
I'm going to mostly disappear for the next ~two months, due to travel and then vacation.

I'll be available on IRC for most of that, but not for coding. I've also been thinking I should change the power-poles to be modular; the next thing I'm likely to push to main Eln is actually a generator/steam turbine combo, but I've been having some trouble getting the thermals right. (It refuses to explode properly.)

(The steam turbine code also involves a TransparentNode variant which supports fluids.. which could be handy for the fuel generator I've seen is partially implemented.)

**Comment by Baughn** [comment]
Er. For IRC, I'm always logged on to Rizon and Freenode. And I think EA needs an official IRC channel. :)

**Comment by Dolu1990** [comment]
+1 for IRC :)

**Comment by Baughn** [comment]
Well... I have a _sort of_ dev server in Erisia, which is on Rizon, so for the time being I've registered #ElectricalAge there.

I know most minecraft channels are on a different network, but Rizon is what I'm familiar with. Anyway, clients do multi-server these days. :)

Maybe put a link to https://qchat2.rizon.net/?channels=ElectricalAge on the website?
